### PR TITLE
Add new ol.geom.Geometry#rotate function

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by fredj** [comment]
Thanks for any review

**Comment by tsauerwein** [comment]
That looks good to me! Maybe a test for `ol.geom.Geometry.prototype.rotate` would be nice, but as you already have solid tests for `ol.geom.flat.transform.rotate` I wouldn't say it's required.

**Comment by ca0v** [comment]
How about doing the same with scale?  See #5684.

File: src/ol/geom/geometry.js [hunk 1]
```diff
@@ -170,6 +170,16 @@ ol.geom.Geometry.prototype.getExtent = function(opt_extent) {
 
 
 /**
+ * Rotate the geometry around a given coordinate. This modifies the geometry
+ * coordinates in place.
+ * @param {number} angle Rotation angle in radians.
```
Comment (tschaub) [inline src/ol/geom/geometry.js]: Always nice to specify the direction (e.g. "Rotation angle in radians (counter-clockwise)").

**Pull request merged**